Feature/joe mai avax

Qkyrie opened the pull request:
- masterchef v3 in the address book
- new mai in the address book
- AMPL in the address book
- joe-ampl-avax (using masterchef v3)
- joe-mai-avax (using masterchef v3)

roman-monk commented:
lets revert package-lock.json and yarn.lock
can use `yarn` to prevent this changes

Qkyrie commented:
> lets revert package-lock.json and yarn.lock can use `yarn` to prevent this changes

reverted lock files. 
Had a talk with cryptomaxi:
- keeping the name of the old LP
- renamed the new LP to -v2
